agents-for-bedrock/features-examples/09-fine-grained-access-permissions/001_avp/template.yaml

```diff
@@ -0,0 +1,182 @@
+                              "attributes": {
+                                  "region": {
+                                      "type": "String",
+                                      "required": true
+                                  },
+                                  "owner": {
+                                      "type": "Entity",
+                                      "name": "User",
+                                      "required": true
+                                  }
+                              }
+                          },
+                          "memberOfTypes": []
+                      },
+                      "Claim": {
+                          "shape": {
+                              "type": "Record",
+                              "attributes": {
+                                  "region": {
+                                      "type": "String",
+                                      "required": true
+                                  },
+                                  "custom-attr-1": {
+                                      "type": "String",
+                                      "required": true
+                                  },
+                                  "owner": {
+                                      "required": true,
+                                      "type": "Entity",
+                                      "name": "User"
+                                  }
+                              }
+                          },
+                          "memberOfTypes": []
+                      },
+                      "User": {
+                          "memberOfTypes": [
+                              "Role"
+                          ],
+                          "shape": {
+                              "attributes": {
+                                  "custom": {
+                                      "attributes": {
+                                          "region": {
+                                              "type": "String",
+                                              "required": false
+                                          }
+                                      },
+                                      "required": false,
+                                      "type": "Record"
+                                  }
+                              },
+                              "type": "Record"
+                          }
+                      }
+                  }
+              }
+          }
+
+  Policy1:
+    Type: AWS::VerifiedPermissions::Policy  
+    Properties:
+      PolicyStoreId: !Ref AvpPolicyStore
+      Definition:
+        Static:
+          Description: 'Allow ClaimsAdministrator role to List claims across all regions'
+          Statement: |
+            permit (
+                principal in avp::claim::app::Role::"ClaimsAdministrator",
+                action in [
+                avp::claim::app::Action::"ListClaims"  
```

Comment:
the claim admin can only list the claims?

Reply:
Yes